pantos/servicenode/blockchains/ethereum.py

```diff
@@ -267,7 +269,7 @@ def cancel_unregistration(self) -> None:
             nonce = node_connections.eth.get_transaction_count(
                 self.__address).get()
             request = BlockchainClient._TransactionSubmissionStartRequest(
-                ContractAbi.PANTOS_HUB,
+                VERSIONED_CONTRACTS_ABI[ContractAbi.PANTOS_HUB],
```

Comment:
This code block is repeating like 3 or 4 times. We could have a function for that, so that we do not need to update the code in different locations. Something like

```python
_submitTransactionStartRequest(*args)
```
What do you think @danut13 ?

Reply:
I looked over it, and I think it's a good idea. However, this block is repeated in different flavours for every function which sends a transaction, and it wouldn't be straight away to make sure that every functionality of the service node works as before after such a change. Therefore, I wouldn't do this in this PR since it's out of scope, but I would create a tech debt ticket

Reply:
Alright, sounds good to me :)

Reply:
Feel free to add it to the ticket. This will make it easier for us to find later.